R/badge.R

```diff
@@ -134,6 +135,22 @@ use_binder_badge <- function(ref = git_default_branch(), urlpath = NULL) {
   invisible(TRUE)
 }
 
+#' @rdname badges
+#' @param cloudurl A link to an existing [RStudio Cloud](https://rstudio.cloud) project
+#' @export
+use_rscloud_badge <- function(cloudurl) {
+  if (grepl("rstudio.cloud/project", cloudurl)) {
+    img <- "https://img.shields.io/badge/launch-cloud-75aadb?style=flat&logo=rstudio"
+    use_badge("Launch RStudio Cloud", cloudurl, img)
+  } else {
+    ui_stop("
+    {ui_code('usethis::use_rscloud_badge()')} requires a link to an existing RStudio Cloud project of the form 'https://rstudio.cloud/project/<id>'.
+    ")
```

Comment:
Can you look around for some other multi-line `ui_stop()`s and handle line breaking and whitespace as they do?

Reply:
Apologies, I used the style currently in use for `ui_oops()` in [`use_badge()`](https://github.com/r-lib/usethis/blame/main/R/badge.R#L43-L47).

In 2abc138c6944cac769edfbf2061e4e4ffbc3edb7, I changed `ui_stop()` to use the style currently used in [`use_github_release()`](https://github.com/r-lib/usethis/blame/main/R/release.R#L177-L179).